java/connector-node/risingwave-sink-jdbc/src/main/java/com/risingwave/connector/JDBCSink.java

```diff
@@ -68,7 +69,18 @@ public JDBCSink(JDBCSinkConfig config, TableSchema tableSchema) {
             // A vector of upstream column types
             List<Integer> columnSqlTypes =
                     Arrays.stream(tableSchema.getColumnNames())
-                            .map(columnTypeMapping::get)
+                            .map(
+                                    columnName -> {
+                                        Integer sqlType = columnTypeMapping.get(columnName);
+                                        if (sqlType != null) {
+                                            return sqlType;
+                                        }
+                                        // Default conversion logic for PostgreSQL data types
+                                        int columnIndex = tableSchema.getColumnIndex(columnName);
+                                        var columnDesc = tableSchema.getColumnDesc(columnIndex);
+                                        return convertRisingWaveTypeToSqlType(
```

Comment:
This seems to be postgres specific. However, `JDBCSink` is not just used by postgres. Will there be an issue?